src/components/icon/icon.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './icon.css';
 
-export default function Icon({ className, name, size, stroke, fill, currentColor, title }) {
+export default function Icon({ className, name, size, stroke, fill, currentColor, title, role, ariaLabel, ariaLabelledBy, a11yKey }) {
```

Comment:
We will want a change log entry with these new props.

Reply:
We will want to add documentation with a more detailed explanation of when to use these. Add an example as well so QA can test it.